libpod/networking_linux.go

```diff
@@ -345,6 +347,11 @@ func (r *Runtime) setupSlirp4netns(ctr *Container) error {
 				}
 			}
 			outboundAddr6 = value
+		case "mtu":
+			mtu, err = strconv.Atoi(value)
+			if mtu < 0 || err != nil {
```

Comment:
The minimum allowed mtu is `68`. Trying to set a value below that will fail with `Error: /usr/bin/slirp4netns failed: "cannot set MTU: Invalid argument\nWARNING: Support for seccomp is experimental\nchild failed(1)\nWARNING: Support for seccomp is experimental\n"`. You should error here if the value is below 68

Reply:
I was hesitant, seeing `create.go` using 0+ as valid.

Reply:
I see we should fix this there as well.